lisa/sut_orchestrator/azure/tools.py

```diff
@@ -55,3 +58,66 @@ def get_generation(self) -> str:
         else:
             generation = "1"
         return generation
+
+
+class LisDriver(Tool):
+    """
+    This is a virtual tool to detect/install LIS drivers.
+    """
+
+    __version_pattern = re.compile(r"^version:[ \t]*([^ \n]*)")
+
+    @property
+    def dependencies(self) -> List[Type[Tool]]:
+        return [Wget, Modinfo]
+
+    @property
+    def command(self) -> str:
+        return "modinfo hv_vmbus"
```

Comment:
We depend on it but not issue it via the tooling (not that I love that syntax...)?

Reply:
yeah I was debating myself if I should use it as a tool or if we make sure modinfo is installed then just use it like command here. Hmm I think I should do otherwise